packages/cli/src/sso/saml/saml.service.ee.ts

```diff
@@ -53,7 +61,7 @@ export class SamlService {
 	}
 
 	constructor() {
-		this.loadSamlPreferences()
+		this.loadFromDbAndApplySamlPreferences()
```

Comment:
Is it not enough to call loadFromDbAndApplySamlPreferences only in the init method?